concrete-core-fixture/src/fixture/plaintext_vector_creation.rs

```diff
@@ -0,0 +1,109 @@
+use crate::fixture::Fixture;
+use crate::generation::prototyping::PrototypesPlaintextVector;
+use crate::generation::synthesizing::SynthesizesPlaintextVector;
+use crate::generation::{IntegerPrecision, Maker};
+use crate::raw::generation::RawUnsignedIntegers;
+use crate::raw::statistical_test::assert_noise_distribution;
+use concrete_commons::dispersion::Variance;
+use concrete_commons::parameters::PlaintextCount;
+
+use concrete_core::prelude::{PlaintextVectorCreationEngine, PlaintextVectorEntity};
+
+/// A fixture for the types implementing the `PlaintextVectorCreationEngine` trait.
+pub struct PlaintextVectorCreationFixture;
+
+#[derive(Debug)]
+pub struct PlaintextVectorCreationParameters {
+    count: PlaintextCount,
+}
+
+impl<Precision, Engine, PlaintextVector> Fixture<Precision, Engine, (PlaintextVector,)>
+    for PlaintextVectorCreationFixture
+where
+    Precision: IntegerPrecision,
+    Engine: PlaintextVectorCreationEngine<Precision::Raw, PlaintextVector>,
+    PlaintextVector: PlaintextVectorEntity,
+    Maker: SynthesizesPlaintextVector<Precision, PlaintextVector>,
+{
+    type Parameters = PlaintextVectorCreationParameters;
+    type RepetitionPrototypes = ();
+    type SamplePrototypes = (Vec<Precision::Raw>,);
+    type PreExecutionContext = (Vec<Precision::Raw>,);
+    type PostExecutionContext = (PlaintextVector,);
+    type Criteria = (Variance,);
+    type Outcome = (Vec<Precision::Raw>, Vec<Precision::Raw>);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![PlaintextVectorCreationParameters {
+                count: PlaintextCount(500),
```

Comment:
Here we should add a plaintextcount of 1 too.

Reply:
Ah, is that the "trivial" case that you guys were discussing?

Reply:
Yes :)